apps/frontend/src/shared/i18n/locales/en/tasks.json

```
@@ -62,6 +62,8 @@
     "pr_created": "PR Created"
   },
   "kanban": {
+    "refreshing": "Refreshing...",
+    "refreshTasks": "Refresh Tasks",
```

Biome (2.1.2)

[error] The key refreshTasks was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)